sample-lnd.conf

```diff
@@ -207,18 +207,20 @@ bitcoin.node=btcd
 ; active.
 ; litecoin.active=1
 
-; Use Bitcoin's test network.
+; Use Litecoin's test network.
 ; litecoin.testnet=1
 ; 
-; Use Bitcoin's simulation test network
+; Use Litecoin's simulation test network
```

Comment:
We might consider removing these, as we currently don't have an ltc simnet.

Reply:
Fixed.